source/reference/mongosync-states.txt

```diff
@@ -0,0 +1,74 @@
+.. _c2c-states:
+
+====================
+``mongosync`` States
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+``mongosync`` enters different states depending on the requests it
```

Comment:
Consider: 
``mongosync`` enters different states depending on the requests it
receives. ``mongosync`` can only be in a single state at a particular time. 
The current ``mongosync`` state determines which API operations you can use.

This page describes the ``mongosync`` states.